### PR TITLE
Updated home page

### PR DESCRIPTION
**Pull request by mikewalch**
I am very open to changes to any of the wording, adding/remove major features, etc.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      </div>
+      <div class="col-md-6">
+        <h4>Cell-based access control</h4>
+        <p>Every key/value pair in Accumulo has its own security label which limits query results based off user authorizations.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-6"> 
+        <h4>Designed to scale</h4>
+        <p>Accumulo is designed to run on a cluster. Additional nodes can be added as the amount data stored in Accumulo grows.</p>
```
Comment (keith-turner) [inline index.html]: Could possibly work in something about running across multiple DFS instances.

**Comment by milleruntime** [comment]
+1 Well summarized bullet points and I like dropping the word "novel".

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      </div>
+      <div class="col-md-6">
+        <h4>Cell-based access control</h4>
+        <p>Every key/value pair in Accumulo has its own security label which limits query results based off user authorizations.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-6"> 
+        <h4>Designed to scale</h4>
+        <p>Accumulo runs on a cluster using one or more HDFS instances. Additional nodes can be added as the amount data stored in Accumulo grows.</p>
```
Comment (keith-turner) [inline index.html]: The second sentence is missing an `of`.

Comment (keith-turner) [inline index.html]: Nodes can be added or removed when as the data size changes.

**Pull request merged**